tests/general/LicenseCheckBearTest.py

```diff
@@ -0,0 +1,45 @@
+import os
+from queue import Queue
+
+from bears.general.LicenseCheckBear import LicenseCheckBear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.results.Result import Result
+from coalib.settings.Section import Section
+
+
+def get_testfile_path(name):
+    return os.path.join(os.path.dirname(__file__),
+                        'licensecheck_test_files',
+                        name)
+
+
+def load_testfile(name):
+    return open(get_testfile_path(name)).readlines()
+
+
+@generate_skip_decorator(LicenseCheckBear)
+class LicenseCheckBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.uut = LicenseCheckBear(Section('name'), Queue())
+
+    def test_license(self):
+        file_contents = load_testfile('mit_license.py')
+        self.check_results(
+            self.uut,
+            file_contents,
+            [],
+            filename=get_testfile_path('mit_license.py'),
+            settings={'licensecheck_lines': 'int = 70',
```

Comment:
Do the tests fail on Ubuntu/Debian if you dont provide `licensecheck_lines` and/or `licensecheck_tail`?

I can report that these two tests pass on Fedora. \o/